pyslim/slim_tree_sequence.py

```diff
@@ -348,8 +348,12 @@ def recapitate(self, recombination_rate=None, keep_first_generation=False,
         if recombination_rate is not None:
             if recombination_map is not None:
                 raise ValueError("Cannot specify length/recombination_rate along with a recombination map")
-            recombination_map = msprime.RecombinationMap(positions = [0.0, self.sequence_length],
-                                                         rates = [recombination_rate, 0.0])
+            if hasattr(msprime.RecombinationMap, "discrete"):
```

Comment:
This is a good interrim solution before msprime 1.0 is released, but we should pin the msprime dep to >=1.0 once it's released and remove it.